### PR TITLE
style: modify interface check way.

### PR DESCRIPTION
**Pull request by L2ncE**
#### What type of PR is this?

style

#### Check the PR title.

- [x] This PR title match the format: \<type\>(optional scope): \<description\>
- [x] The description of this PR title is user-oriented and clear enough for others to understand.

#### Translate the PR title into Chinese.

修改接口检查的方式。

#### More detail description for this PR(en: English/zh: Chinese).

en: For more details, see #346 .

zh: 更多信息，请详见 #346 。

#### Which issue(s) this PR fixes:

#346

### CODE REVIEW

**Comment by demoManito** [comment]
pkg/common/tracer/traceinfo/httpstats.go

**Comment by L2ncE** [comment]
> Interface check in `pkg/common/tracer/traceinfo/httpstats.go` not modified

I got it, thx.

File: pkg/common/tracer/traceinfo/httpstats.go [hunk 1]
```diff
@@ -24,7 +24,7 @@ import (
 )
 
 var (
-	_           HTTPStats = &httpStats{}
+	_           HTTPStats = (*httpStats)(nil)
```
Comment (demoManito) [inline pkg/common/tracer/traceinfo/httpstats.go:27]: It is recommended not to put the following variables together

`var _ HTTPStats = (*httpStats)(nil)`

Comment (L2ncE) [reply pkg/common/tracer/traceinfo/httpstats.go]: > It is recommended not to put the following variables together
> 
> `var _ HTTPStats = (*httpStats)(nil)`

done

**Review by Duslia — APPROVED** [review]

**Review by welkeyever — APPROVED** [review]
lgtm

**Pull request merged**